hack/ocm-minikube.sh

```diff
@@ -8,6 +8,9 @@ set -x
 set -e
 trap 'echo exit value: $?;trap - EXIT' EXIT
 
+# pre-requisites
+which jq
```

Comment:
@hatfieldbrian if you prefer a different means to test tool availability do let me know.

Reply:
I prefer [POSIX's](https://pubs.opengroup.org/onlinepubs/9699919799/utilities/command.html) `command -v`, but `which` should work.

Reply:
jFYI, I had fixed this to `command -v ...` as shellcheck caught the same.